Sources/Orbit/Components/Slider.swift

```diff
@@ -0,0 +1,441 @@
+import SwiftUI
+
+public typealias SliderValue = BinaryFloatingPoint & Strideable & CustomStringConvertible
+
+/// Enables selection from a range of values.
+///
+/// - Note: [Orbit definition](https://orbit.kiwi/components/interaction/slider/)
+public struct Slider<Value>: View where Value: SliderValue, Value.Stride: BinaryFloatingPoint {
+
+    enum SelectedValue {
+        case single(Binding<Value>?)
+        case range(Binding<ClosedRange<Value>>?)
+    }
+
+    @Environment(\.sizeCategory) var sizeCategory
+
+    let value: SelectedValue
+    let valueLabel: String
+    let startLabel: String
+    let endLabel: String
+    let bounds: ClosedRange<Value>
+
+    @State private var isFirstThumbDrag = false
+    @State private var isSecondThumbDrag = false
+    @State private var firstValue: Value {
+        didSet {
+            switch value {
+                case .single(let currentValue):
+                    currentValue?.wrappedValue = firstValue
+                case .range:
+                    updateSelectedRangeIfNeeded()
+            }
+        }
+    }
+
+    @State private var secondValue: Value? {
+        didSet {
```

Comment:
Do these `didSet` work? AFAIK this is not supposed to be working 🤔 

https://forums.swift.org/t/state-vars-didset-fix-or-prohibition/53161

https://developer.apple.com/forums/thread/123357

Reply:
From your first link:
>  So it is somewhat expected in that perspective for the didSet to only trigger on direct mutations of the property itself (and not through it's projection).

Also here https://forums.swift.org/t/using-error-protocol-in-swiftui-binding-does-any-or-some-fix-this/62740/4:
> The reason is that this didSet will trigger only via the computed error.set accessor and the Binding generated by State does not pass the value through that. It does only work if the view sets the property directly (e.g. error = someError), not through a binding.

This is our case.